src/main/java/uk/gov/pay/directdebit/payments/dao/GoCardlessEventDao.java

```diff
@@ -14,10 +14,43 @@
 @RegisterRowMapper(GoCardlessEventMapper.class)
 public interface GoCardlessEventDao {
 
-    @SqlUpdate("INSERT INTO gocardless_events(internal_event_id, event_id, action, resource_type, json, created_at) VALUES (:eventId, :goCardlessEventId, :action, :resourceType, CAST(:json as jsonb), :createdAt)")
+    @SqlUpdate("INSERT INTO gocardless_events(internal_event_id, event_id, action, resource_type, json," +
```

Comment:
There is a test in `GoCardlessDirectDebitEventDaoIT.shouldInsertAnEvent` which should be extended to assert that the new values are being inserted and retrieved correctly.